auth/user_mgt.go

```diff
@@ -525,27 +602,45 @@ func makeExportedUser(r responseUserRecord) (*ExportedUserRecord, error) {
 		}
 	}
 
+	var providerID string
+	if len(r.ProviderUserInfo) == 1 {
+		providerID = r.ProviderUserInfo[0].ProviderId
```

Comment:
@avishalom can you verify this logic is correct? Looks a little strange.

Reply:
It's a mistake, i will fix it. 
This is Because on the UserRecord, the UserInfo struct is used for ProviderUserInfo and for  UserInfo. But providerID is only set for a ProviderUserInfo.

````
// UserRecord contains metadata associated with a Firebase user account.
type UserRecord struct {
	*UserInfo
	CustomClaims     map[string]interface{}
	Disabled         bool
	EmailVerified    bool
	ProviderUserInfo []*UserInfo
	UserMetadata     *UserMetadata
}
```

Reply:
Does it make sense to add a test that would have failed in this case?